settings/DevHome.Settings/Views/ExtensionSettingsPage.xaml

```diff
@@ -0,0 +1,43 @@
+<!-- Copyright (c) Microsoft Corporation and Contributors. -->
+<!-- Licensed under the MIT License. -->
+
+<Page
+    x:Class="DevHome.Settings.Views.ExtensionSettingsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:views="using:DevHome.Common.Views"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never"
+    mc:Ignorable="d">
+
+    <Grid
+    MaxWidth="{ThemeResource MaxPageContentWidth}"
+    Margin="{ThemeResource ContentPageMargin}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <BreadcrumbBar
+        x:Name="BreadcrumbBar"
+        Margin="0,0,0,16"
+        ItemClicked="BreadcrumbBar_ItemClicked"
+        ItemsSource="{x:Bind ViewModel.Breadcrumbs}" />
+
+        <ScrollViewer Grid.Row="1" VerticalAlignment="Top">
+            <StackPanel>
+                <views:PluginAdaptiveCardPanel x:Name="SettingsContent">
+                    <i:Interaction.Behaviors>
+                        <ic:EventTriggerBehavior EventName="Loaded">
+                            <ic:InvokeCommandAction Command="{x:Bind ViewModel.SettingsContentLoadedCommand}" CommandParameter="{x:Bind SettingsContent, Mode=OneWay}" />

```

Comment:
I'm getting this warning, the github build might complain more than VS:
```2>R:\devhome\settings\DevHome.Settings\Views\ExtensionSettingsPage.xaml(36,111): XamlCompiler warning WMC1506: OneWay bindings require at least one of their steps to support raising notifications when their value changes```

I think just take out the Mode so it defaults to OneTime, since it's never going to change.